packages/generator/src/parser/v4/edmx-to-vdm.ts

```diff
@@ -28,7 +28,9 @@ function navigationProperties(
   const entityType = entityMetadata.entityType as EdmxEntityType;
   const entitySet = entityMetadata.entitySet as EdmxEntitySet;
 
-  return entitySet.NavigationPropertyBinding.map(navBinding => {
+  return entitySet.NavigationPropertyBinding.filter(
+    navBinding => !navBinding.Path.includes('/')
```

Comment:
cloud you wrap the check with the `includes(/)` in a helper function with a specing name like `excludeNavigationalPropoertiesOfDerivedTypes` or something similar so that one has a better understanding what this is about. Also add then a TODO that this will be removed after we come up with an idea how to handle this topic properly.

Reply:
✅